DeezyMatch/candidateRanker.py

```diff
@@ -47,8 +50,13 @@ def candidate_ranker(input_file_path="default", scenario=None, ranking_metric="f
     
     # read input file
     dl_inputs = read_input_file(input_file_path)
+
+
+    if (ranking_metric.lower() in ["faiss"]) and (selection_threshold < 0):
+        sys.exit(f"[ERROR] Threshold for the selected metric: '{ranking_metric}' should be >= 0.")
+    if (ranking_metric.lower() in ["cosine", "conf"]) and not (0 <= selection_threshold <= 1):
```

Comment:
Silly comment, but what happens if the metric is not a known one, like they type "fais" - could you add a small catch that says: metric is currently not supported - wee support only faiss, cosine, conf

Reply:
Done: https://github.com/Living-with-machines/DeezyMatch/pull/68/commits/dc5bd085bf44401c6710aff94992b2e967127a78